staff/roberto-perez/e-commerce-template/style.css

```diff
@@ -0,0 +1,146 @@
+html {
+    box-sizing: border-box;
+}
+html, body {
+    margin: 0px;
+    padding: 0px;
+}
+body {
+    font-family: 'Poppins', sans-serif;
+}
+#header {
+    background-color: rgb(27, 29, 30);
+    display: flex;
+    align-items: center;
+    z-index: 3;
+    top: 0px;
+    position: sticky;
+    justify-content: space-between;
+    padding: 10px 30px;
+}
+#header nav ul {
+    display: none;
+    margin: 0px;
+    padding: 0px;
+}
+#header nav ul li {
+    list-style: none;
+    display: inline-block;
+    margin-left: 10px;
+}
+#header nav ul li a {
+    color: #fff;
+    text-decoration: none;
+}
+#header nav ul li a:hover {
+    color: #70acec;
+}
+@media (min-width: 768px) {
+    #header nav ul {
+        display: block;
+    } 
+}
+button.bars {
+    color: rgb(255, 255, 255);
+    font-size: 1.2em;
+    background: none;
+    border: 0px;
+    outline: 0px;
+    cursor: default;
+    padding: 1px 7px 2px;
+}
+@media (min-width: 768px) {
+    button.bars {
```

Comment:
here you don't need to write button.bars, to point to the button with class bars, you can directly write .bars

Reply:
Done!